ledfx/effects/melt_and_sparkle.py

```diff
@@ -62,7 +62,7 @@ class MeltSparkle(AudioReactiveEffect, HSVEffect):
                 "strobe_blur",
                 description="How much to blur the strobes",
                 default=3.5,
-            ): vol.All(vol.Coerce(float), vol.Range(min=0, max=10)),
+            ): vol.All(vol.Coerce(float), vol.Range(min=0.1, max=10)),
```

Comment:
I think it's probably nice to be able to have no blur on the strobes.

How about instead of making the min 0.1, we check for 0 further down and only call smooth if the blur is > 0?

That way we can have crisp strobes if we want, but prevent blowing up smooth with a sigma of 0.

Reply:
And if we alter smooth to just return the array as is when sigma = 0, we don't need to do anything at all here, we can leave min = 0 and just let smooth figure it out.